assets/js/googlesitekit/datastore/user/index.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * core/user data store
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import authentication from './authentication';
+import disconnect from './disconnect';
+import { STORE_NAME } from './constants';
+
+export const INITIAL_STATE = Data.collectState(
+	authentication.INITIAL_STATE,
+	disconnect.INITIAL_STATE
+);
+
+export { STORE_NAME };
```

Comment:
Let's keep this as the first export (I believe all stores currently do this). It can go right below the import 👍